src/main/java/com/refinedmods/refinedstorage/item/blockitem/ControllerBlockItem.java

```diff
@@ -3,10 +3,32 @@
 import com.refinedmods.refinedstorage.RS;
 import com.refinedmods.refinedstorage.api.network.NetworkType;
 import com.refinedmods.refinedstorage.block.ControllerBlock;
+import com.refinedmods.refinedstorage.util.BlockUtils;
+import net.minecraft.block.Block;
+import net.minecraft.item.DyeColor;
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.text.IFormattableTextComponent;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.fml.RegistryObject;
 
 public class ControllerBlockItem extends EnergyBlockItem {
-    public ControllerBlockItem(ControllerBlock block) {
+    private final IFormattableTextComponent displayName;
+
+    public ControllerBlockItem(ControllerBlock block, DyeColor color, RegistryObject<? extends Block> blockForTranslation) {
```

Comment:
Just take `ITextComponent` here instead of the entire `RegistryObject<? extends Block> blockForTranslation`